ledger/participant-integration-api/src/main/scala/platform/store/dao/LedgerDao.scala

```diff
@@ -167,6 +168,7 @@ private[platform] trait LedgerReadDao extends ReportsHealth {
 }
 
 private[platform] trait LedgerWriteDao extends ReportsHealth {
+  def transactionsWriter: TransactionsWriter
```

Comment:
Why do we need this to become part of the interface?

Reply:
This was needed so it can be called from someone else than its owner

Reply:
It's too bad I couldn't go on and completely break `JdbcLedgerDao` apart. Its interface is way too wide. Thanks for clarifying.